packages/runtime/container-runtime/src/summary/documentSchema.ts

```diff
@@ -550,6 +621,13 @@ export class DocumentsSchemaController {
 		// features that are mentioned in schema right away, without a need to go through schema transition (and thus for a session or
 		// two losing ability to use all the features)
 
+		// TODO: This is a hacky way to ensure we always use greater of the two minVersionForCollab. It would be better to avoid
+		// custom logic here.
+		this.documentSchema.minVersionForCollab = getGreaterMinVersionForCollab(
```

Comment:
Yeah this shouldn't be necessary. Once `maybeSendSchemaMessage` happens and we process the ack via `processDocumentSchemaMessages`, documentSchema updates to include the new value passed in by this session (if it's greater than the old value)

Reply:
Ah you're right. My tests were missing the step of calling `processDocumentSchemaMessages` manually